src/java/org/apache/cassandra/index/sai/disk/format/Version.java

```diff
@@ -38,15 +39,18 @@
     public static final Version AA = new Version("aa", V1OnDiskFormat.instance, Version::aaFileNameFormat);
     // Stargazer
     public static final Version BA = new Version("ba", V2OnDiskFormat.instance, (c, i) -> stargazerFileNameFormat(c, i, "ba"));
+    // ?
+    public static final Version CA = new Version("ca", V3OnDiskFormat.instance, (c, i) -> stargazerFileNameFormat(c, i, "ca"));
+
 
     // These are in reverse order so that the latest version is used first. Version matching tests
     // are more likely to match the latest version so we want to test that one first.
-    public static final List<Version> ALL = Lists.newArrayList(AA, BA);
+    public static final List<Version> ALL = Lists.newArrayList(AA, BA, CA);
 
     public static final Version EARLIEST = AA;
     // The latest version can be configured to be an earlier version to support partial upgrades that don't
     // write newer versions of the on-disk formats.
-    public static final Version LATEST = parse(System.getProperty("cassandra.sai.latest.version", "ba"));
+    public static final Version LATEST = parse(System.getProperty("cassandra.sai.latest.version", "ca")); // parse("ca");
```

Comment:
nit: I'm not sure of the benefit of the comment. It isn't correct if `cassandra.sai.latest.version` has been set.